src/Operator/Binary/FunctionBinaryOperator.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Twig\Operator\Binary;
+
+use Twig\Attribute\FirstClassTwigCallableReady;
+use Twig\Error\SyntaxError;
+use Twig\ExpressionParser;
+use Twig\Node\EmptyNode;
+use Twig\Node\Expression\AbstractExpression;
+use Twig\Node\Expression\MacroReferenceExpression;
+use Twig\Node\Expression\NameExpression;
+
+class FunctionBinaryOperator extends AbstractBinaryOperator
+{
+    private $readyNodes = [];
+
+    public function parse(ExpressionParser $parser, AbstractExpression $expr, int $line): AbstractExpression
+    {
+        if (!$expr instanceof NameExpression) {
+            throw new SyntaxError('Function name must be a string.', $line, $parser->getStream()->getSourceContext());
```

Comment:
```suggestion
            throw new SyntaxError('Function name must be an identifier.', $line, $parser->getStream()->getSourceContext());
```

In terms of syntax, a string would have quotes around it.

Reply:
Correct